### ISSUE TITLE
Can the compiler folder of the cli-serve plugin be editable?

### ISSUE THREAD

**Issue by leo-wj910729** [vuejs/vue-cli#1888]
### What problem does this feature solve?
I used the webpack template in vue-cli version 2.0. This template can be used to configure folders and files. I may need to create a new project soon at work, so I need to have some public files in the template. Reducing the time I spent creating projects, I know that I can now create a preset.json file to set some of the configuration items, but there are no instructions in the documentation for keeping files.

### What does the proposed API look like?
类似于这样的项目公用文件,希望可以进行配置
## 文件目录
```
src
├── assets          
    ├── sass        
    ├── less        
    └── img         
├── components      
    ├── Login       
    ├── Loading     
    ├── Marquee     
├── plugins         
    ├── axios.js            
    ├── store.js            
    └── router.js   
```

**Comment by yyx990803** [comment]
Maybe we should allow a preset to access the plugin API - making it a plugin itself?

**Comment by leo-wj910729** [comment]
@yyx990803 For those who use the webpack template in the vue2.0 version, there is such a plugin that is really good.

**Comment by leo-wj910729** [comment]
@Akryum Can I override the default file when I use the plugin to create a file? For example, use cli to create a project and use a router, but I don't want him to create it by default path, I hope I can decide

**Comment by sodatea** [comment]
Closed as support for generator.js and prompts.js in preset has been implemented since v3.0.0-rc.12
